.github/workflows/image.yaml

```diff
@@ -49,6 +50,9 @@ jobs:
           - ispr: true
             dist: ubuntu20.04
             driver: 550.127.05
+          - ispr: true
+            dist: ubuntu20.04
+            driver: 565.57.01
```

Comment:
Question -- are we able to replace the three exclude bullets with just
```
- ispr: true
  dist: ubuntu20.04
```
since we are effectively skipping ubuntu20.04 on PRs?

Reply:
tbh, I just added it for the sake of adding an excludes directive. I don't think we need to exclude add ubuntu20.04 dists